lib/content/message/predictions.ex

```diff
@@ -77,6 +79,14 @@ defmodule Content.Message.Predictions do
         true -> predicted_time |> Kernel./(60) |> round()
       end
 
+    crowding_data_confidence =
+      calculate_crowding_data_confidence(
+        prediction,
+        Engine.Locations.for_vehicle(prediction.vehicle_id)
```

Comment:
Yeah that's a good point. I did end up thinking about that later down the line in a later PR and changed the logic so that we only fetch location if it's an Orange Line prediction. Take a look at [this logic](https://github.com/mbta/realtime_signs/pull/677/files#diff-ad43236a6e7bf15123bee1479271a49d884cf1da41b61a88d5a19e38ccc69d6eR129) for `do_crowding/1` and let me know what you think

Reply:
That does improve things. And the lookup should be fast enough, so it's probably not worth worrying about.